OpenRA.Mods.Common/Traits/Render/WithCrateBody.cs

```diff
@@ -74,7 +79,9 @@ void INotifyParachuteLanded.OnLanded(Actor ignore)
 
 		void PlaySequence()
 		{
-			var sequence = self.World.Map.GetTerrainInfo(self.Location).IsWater ? info.WaterSequence : info.LandSequence;
+			var cell = self.Location;
+			onWater = self.World.Map.Contains(cell) && info.WaterTerrainTypes.Contains(self.World.Map.GetTerrainInfo(cell).Type);
```

Comment:
Originally I wanted to make sure the sequence is only played when the cell is in the visible map area. But to be honest, I'm not sure such a safeguard is really needed, and I see now that this would be the wrong place to put it anyway, as it would just switch to LandSequence instead. I guess I'll just remove it.